### PR TITLE
Adding schema for landing page details

### PR DESCRIPTION
**Pull request by aarjain**
Please link to the issue #…

### CODE REVIEW

File: extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/landingpage/name": {
+          "title": "Landing Page Name",
+          "type": "string",
+          "description": "Name of Landing Page."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/landingpage/inputDetails": {
+          "title": "Landing Page Input",
+          "type": "object",
+          "meta:xdmType": "map",
```
Comment (mjasrotia) [inline extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json:32]: "Map" types are generally not supported very well through Query Services and generally advised not to be used. Do you folks still want this field to be marked as "map" type ?

File: extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          "description": "Details Input by user in the landing page"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/landingpage/additionalContext": {
+          "title": "Landing Page Additional Context",
+          "type": "object",
+          "meta:xdmType": "map",
```
Comment (mjasrotia) [inline extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json:38]: "Map" types are generally not supported very well through Query Services and generally advised not to be used. Do you folks still want this field to be marked as "map" type ?

File: extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          "items": {
+            "type": "object",
+            "properties": {
+              "https://ns.adobe.com/experience/customerJourneyManagement/landingpage/subscriptions/id": {
+                "title": "Subscription ID",
+                "type": "string",
+                "description": "Unique Identifier for Subscription"
+              },
+              "https://ns.adobe.com/experience/customerJourneyManagement/landingpage/subscriptions/isOptIn": {
+                "title": "Opt in or opt out a subscription list",
```
[the remark this answers is not in the document]
Comment (aarjain) [reply extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json]: Made the change

File: extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json (hunk 1 above)
Comment (aarjain) [reply extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json:37]: @mjasrotia : We currently need this schema for creating a rule based event in journey and for message personalization for the message that will be sent in that journey. The schema is not fixed as different customers may want different data in the landing page form and we need to send that information in this event.
CC @sawhneyh

Comment (mjasrotia) [reply extensions/adobe/experience/customerJourneyManagement/landingpage.schema.json:37]: Sounds good. I just wanted to make sure you are aware of the limitations.  Anyways, the way the map properties were defined was wrong. I changed the definition. Please note the change for future references.

**Review by saurabh111091 — APPROVED** [review]
+1
Can probably change ""meta:status": "stable" to "experimental"

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**